packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -540,7 +538,7 @@ class DebuggerController extends DisposableController
         .map(
           (v) => DartObjectNode.create(
             v,
-            serviceManager.appState.isolateRef.value,
+            serviceManager.isolateManager.selectedIsolate.value,
```

Comment:
optional - we could add a convenience getter to service manager:
`IsolateRef? get selectedIsolate => isolateManager.selectedIsolate.value;` and use that everywhere for brevity